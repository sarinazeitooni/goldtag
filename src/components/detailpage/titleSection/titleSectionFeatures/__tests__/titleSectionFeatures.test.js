import React from 'react';
import { shallow } from 'enzyme';
import TitleSectionFeatures from "../titleSectionFeatures";
describe("TitleSectionFeatures component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<TitleSectionFeatures id={{items : ['','']}} />)
        expect(container).toMatchSnapshot();
    });
});