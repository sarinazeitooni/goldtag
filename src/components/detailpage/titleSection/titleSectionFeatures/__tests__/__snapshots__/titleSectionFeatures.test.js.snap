// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleSectionFeatures component it should render a div 1`] = `ShallowWrapper {}`;
